.github/workflows/jekyll-gh-pages.yml

```
@@ -0,0 +1,68 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: - elparadisogonzalo Setup Go environment
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 2: block sequence entries are not allowed in this context

(syntax-check)